### PR TITLE
mgmt, update dns and containerregistry

### PR DESCRIPTION
**Pull request by weidongxu-microsoft**
regen only (mostly for sample generation)
- resources
- keyvault
- private dns
-  msi
- containerinstance

update
- dns to 2018-05-01
- containerregistry to 2021-09-01
live tests pass for updated RPs

major breaking change happen to containerregistry, as stable version removed a few client from previous preview version

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](../../CONTRIBUTING.md#building-and-unit-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: sdk/resourcemanager/azure-resourcemanager-containerregistry/CHANGELOG.md [hunk 1]
```diff
@@ -2,14 +2,16 @@
...
-### Features Added
-
 ### Breaking Changes
 
-### Bugs Fixed
+- Removed preview features of `ExportPipelinesClient`, `ImportPipelinesClient`, `PipelineRunsClient`, `ScopeMapsClient`, `TokensClient`.
 
 ### Other Changes
 
+### Dependency Updates
```
Comment (haolingdong-msft) [inline sdk/resourcemanager/azure-resourcemanager-containerregistry/CHANGELOG.md:11]:
```suggestion
#### Dependency Updates
```

File: sdk/resourcemanager/azure-resourcemanager-dns/CHANGELOG.md [hunk 2]
```diff
@@ -2,13 +2,11 @@
 
 ## 2.11.0-beta.1 (Unreleased)
 
-### Features Added
-
-### Breaking Changes
+### Other Changes
 
-### Bugs Fixed
+### Dependency Updates
```
Comment (haolingdong-msft) [inline sdk/resourcemanager/azure-resourcemanager-dns/CHANGELOG.md:7]:
```suggestion
#### Dependency Updates
```

**Review by haolingdong-msft — APPROVED** [review]

**Comment by dol** [comment]
@weidongxu-microsoft In the update the preview version and some corresponding clients like `ScopeMapsClient` were removed. At the moment I'm planing of using the REST endpoint for ScopeMaps, but the SDK doesn't support it right now.
Should I use an older version like `2.10.0` to work with the preview API? Or are there somewhere dedicated release with preview support?

**Comment by weidongxu-microsoft** [comment]
@dol 

Very sorry for the confusing behavior. In principle, the stable release like 2.10.0 should not use or include preview api-version of service (e.g. the ScopeMaps). However this case (if I remember correctly) with preview api-version in 2.10.0 (maybe a few earlier versions) is necessitated by private link feature.

For the time being, you can continue use the old 2.10.0 SDK, if your task does not require any new feature.
If you do require a new SDK with ScopeMaps (but it still in preview api-version), you can request a beta release from us. Just create a new issue for it (mentioning you require preview feature of ACR) and we can release e.g. 2.15.0-beta.1 for you.

**Pull request merged**